UDx-container/Makefile

```diff
@@ -95,9 +95,9 @@ endif
 
 ifeq ($(TARGET), rpm)
   TARGET_U=RPM
-  CONTAINER_OS=almalinux
-  CONTAINER_OS_TAG=alma
-  CONTAINER_OS_VERSION ?= 8
+  CONTAINER_OS=docker-registry.verticacorp.com/opentext/redhat
```

Comment:
They won't be able to without vpn unfortunately. And even if they did want to, Red Hat requires a subscription to make use of their repositories. 

I'm not well versed to the use case of these (sdk/k8,etc) containers. Do end users *need* the ability to build the containers on their own? If so, I don't know if keeping the old base image as a target option will be compliant with OT's security policies. It sort of defeats the purpose of releasing a secure 'hardened' image. 

If end users use the final product as is, never needing to build outside the vpn, then we are good.

On a related note, are there any tests/validations that we can run to make sure the OT hardened image works as expected?

Reply:
I am not sure about their use cases too. I think vertica-sdk is mainly used internally(certainly not on k8s) like in Vertica/dblink repo. 
If no one is expected to build the image then this is fine.

Reply:
It's better not to use the internal registry in the code. Could we change CONTAINER_OS to accept the env variable? Then we can build the image based on our internal registry and release it to dockerhub.

Reply:
@qindotguan good point! I'll create a newly required 'IMAGE' parameter only for rpm builds. This will give end users the option to build outside our vpn using their own images (and not hold us liable). We will just have to know internally which hardened image to use when we release.

Reply:
I suggest the env var's default value to be the current base image.

Reply:
Yup make sense, was confused previously. I'll keep the default to alma and set the Jenkins build to pass in the OT hardened image